site/content/3.11/develop/integrations/spring-data-arangodb/_index.md

```diff
@@ -37,66 +39,131 @@ Spring Data ArangoDB is compatible with:
 
 {{< /tabs >}}
 
-## Maven
+## Get started
 
-To use Spring Data ArangoDB in your project, your build automation tool needs to
-be configured to include and use the Spring Data ArangoDB dependency.
-Example with Maven (substitute `x.x.x` with the latest Spring Data ArangoDB version):
+This tutorial is about how to configure [Spring Data ArangoDB](https://github.com/arangodb/spring-data)
+without using Spring Boot Starter ArangoDB.
 
-```xml
-<dependency>
-  <groupId>com.arangodb</groupId>
-  <artifactId>arangodb-spring-data</artifactId>
-  <version>x.x.x</version>
-</dependency>
-```
+For a more extensive tutorial about the features of Spring Data ArangoDB and
+Spring Boot support, see the [Spring Boot Starter](../spring-boot-arangodb.md)
```

Comment:
I added "and Spring Boot support" in order to remove the separate headline with the paragraph that said "Spring Boot support is offered by Spring Boot Starter ArangoDB" but doesn't this demo use Spring Boot? Doesn't seem quite right